Source/Stationeers.Addons.Patcher/Core/Patchers/MonoPatcher.cs

```diff
@@ -123,23 +117,22 @@ public bool IsPatched()
             return false; // Not patched
         }
 
-        private void DiscoverInstallInstance()
+        /// <inheritdoc />
+        private void GetInstanceAssemblies(String installInstance)
         {
             // This is kind of verbose, might need to be rewritten in a more concise manner
-            if (File.Exists(Path.Combine(Environment.CurrentDirectory, Constants.GameResourcesDir, AssemblyDir, AssemblyName)))
+            String InstallResourcesDir = null;
+            if(installInstance == Constants.GameExe)
             {
-                AssemblyFileName = Path.Combine(Environment.CurrentDirectory, Constants.GameResourcesDir, AssemblyDir, AssemblyName);
-                TemporaryAssemblyFileName = Path.Combine(Environment.CurrentDirectory, Constants.GameResourcesDir, AssemblyDir, AssemblyName + ".temp.dll");
-                Logger.Current.Log($"Discovered game instance via assembly file '{AssemblyFileName}'");
-                return;
+                InstallResourcesDir = Constants.GameResourcesDir;
             }
-            if (File.Exists(Path.Combine(Environment.CurrentDirectory, Constants.ServerResourcesDir, AssemblyDir, AssemblyName)))
+            if(installInstance == Constants.ServerExe)
             {
-                AssemblyFileName = Path.Combine(Environment.CurrentDirectory, Constants.ServerResourcesDir, AssemblyDir, AssemblyName);
-                TemporaryAssemblyFileName = Path.Combine(Environment.CurrentDirectory, Constants.ServerResourcesDir, AssemblyDir, AssemblyName + ".temp.dll");
-                Logger.Current.Log($"Discovered game instance via assembly file '{AssemblyFileName}'");
-                return;
+                InstallResourcesDir = Constants.ServerResourcesDir;
             }
+
```

Comment:
```CS
Debug.Assert(!string.IsNullOrEmpty(installResourcesDir), "Invalid install dir!");
```
Would help a bit :)

Reply:
This gets caught a little down the line in your original code but I'll go ahead and add it

Reply:
Resolved: https://github.com/Erdroy/Stationeers.Addons/pull/3/commits/741f607f74d7a2c0ebbeff7f8f496681b4ac2656